### PR TITLE
Box

### PR DESCRIPTION
**Pull request by AlexandreBroudin**

### CODE REVIEW

File: src/scss/components/box/_header.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.Box-header {
+  padding: $pd--m;
+  overflow: auto;
```
Comment (GabrielCousin) [inline src/scss/components/box/_header.scss]: :tractor:

**Comment by GabrielCousin** [comment]
:pager:

File: src/scss/components/box/_header.scss (hunk 1 above)
Comment (ugogo) [inline src/scss/components/box/_header.scss]: clearfix :warning:

File: src/scss/components/box/_header.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  + .Box-content {
+    padding-top: 0;
+  }
+}
+
+.Box-title {
+  display: inline-block;
+  margin-top: 0;
```
Comment (ugogo) [inline src/scss/components/box/_header.scss]: `margin: 0` ?

File: src/scss/components/box/_header.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.Box-header {
+  padding: $pd--m;
+  overflow: auto;
+
+  + .Box-content {
```
Comment (ugogo) [inline src/scss/components/box/_header.scss]: This block declaration should be in `_content.scss`

File: src/scss/components/box/box.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+      Pellentesque habitant
+    </div>
+  </div>
+  <div class="Box">
+    <div class="Box-content">
+      Pellentesque habitant
+    </div>
+  </div>
+</div>
+<div class="Boxes Boxes--horizontal">
```
Comment (ugogo) [inline src/scss/components/box/box.md]: L E G A C Y ? `Boxes--horizontal`

**Comment by ugogo** [comment]
- What about a `.Box` follow by another `.Box`?
- Remove `website/components/box.html` since it won't be tracked

File: src/scss/components/box/box.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+      <h4 class="Box-title">Title</h4>
+    </div>
+    <div class="Box-content">
+      Pellentesque habitant
+    </div>
+  </div>
+</div>
+```
+
+## Floated content
```
Comment (ugogo) [inline src/scss/components/box/box.md]: Is this really useful, since it's only a layout case?

File: src/scss/components/box/box.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+      <a class="" href="#">Baz</a>
+    </nav>
+  </div>
+  <div class="Box-content">
+    <p> Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+    Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.
+    Donec eu libero sit amet quam egestas semper.
+    Aenean ultricies mi vitae est. Mauris placerat eleifend leo.</p>
+  </div>
+  <table class="Table Table--actionnable">
```
Comment (ugogo) [inline src/scss/components/box/box.md]: Why this table isn't in `Box-content`?
And same as https://github.com/Wisembly/tapestry/pull/138/files#r39514273, is this really useful?

**Pull request merged**